packages/rum/test/e2e/general-usecase/app.e2e-spec.js

```diff
@@ -129,4 +79,26 @@ describe('general-usercase', function() {
 
     return allowSomeBrowserErrors(['timeout test error with a secret'])
   })
+
+  it('should capture history.pushState', function() {
+    /**
+     * The query string is only used to make url different to the previous test,
+     * Otherwise, both tests will run in the same window.
+     */
+
+    browser.url('/test/e2e/general-usecase/index.html?run=pushState#test-state')
+    browser.waitUntil(
+      () => {
+        return $('#test-element').getText() === 'Passed'
+      },
+      5000,
+      'expected element #test-element'
+    )
+
+    const serverCalls = waitForApmServerCalls(0, 1)
+    expect(serverCalls.sendTransactions.length).toBe(1)
+    const transactionPayload = serverCalls.sendTransactions[0].args[0][0]
+    expect(transactionPayload.name).toBe('Push state title')
+    expect(transactionPayload.spans.length).toBeGreaterThanOrEqual(3)
```

Comment:
can we document what type of spans we can see here?.

Reply:
Added comments to clarify!